concourse/pipelines/create-deployer.yml

```diff
@@ -467,6 +467,10 @@ jobs:
                               ./paas-cf/manifests/concourse-manifest/extensions/datadog-agent.yml
                               ./paas-cf/manifests/shared/deployments/datadog-agent.yml"
             fi
+            # shellcheck disable=SC2050
```

Comment:
That approach is possible, but it requires many more changes. Aim in this story was to quickly implement something and see how it does. If we want to have more changes (e.g. different timeouts for CI & staging), then we'll create another story. In that case, we can create 4 env. specific files (dev, ci, staging, prod) and always spruce them in. Currently I prefer concourse manifest to be simple and mostly contained in a single file only.